cardano_node_tests/tests/tests_plutus/test_mint_build.py

```diff
@@ -1198,3 +1199,16 @@ def test_duplicated_collateral(
             utxos=out_utxos, address=issuer_addr.address, coin=token
         )
         assert token_utxo and token_utxo[0].amount == token_amount, "The token was NOT minted"
+
+        # check return collateral amount
+
+        collateral_amount = collateral_utxos[0].amount
```

Comment:
Optional: It is not necessary to have this stored in variable. It is used just once and meaning of `collateral_utxos[0].amount` is IMO straightforward.

Reply:
it was just to be super readable on the assertion, but I made the update :+1: